middle_end/flambda/from_lambda/prepare_lambda.mli

```diff
@@ -20,6 +20,4 @@
     handlers which are to be treated as recursive.  (This is rather more
     straightforward than changing the type in [Lambda] to accommodate
     this). *)
```

Comment:
Could you update the comment too ? Or remove it completely, as it only documents the recursive continuations.